.github/workflows/main.yml

```diff
@@ -61,35 +61,21 @@ jobs:
       - name: Build Docker Image
         if: github.ref != 'refs/heads/main'
         run: |
-          docker build -t "$REGISTRY_HOSTNAME"/"$HOST"/"$IMAGE":${{ steps.tag.outputs.pr_number }} -f _infra/docker/Dockerfile .
```

Comment:
I think we need to keep using the docker images in _infra directory as they are the ones with gunicorn